common.py

```diff
@@ -182,6 +183,10 @@ def clone_repos():
         '{}/llvm-project '.format(
             branches[swift_branch]['llvm-project'], workspace
         ),
+        '{} git@github.com:apple/swift-llvm-bindings.git'
```

Comment:
You will need to add a space here so the git clone command doesn't thing the whole string is the repository name

```suggestion
        '{} git@github.com:apple/swift-llvm-bindings.git '
```

Reply:
Fixed, thanks for noticing!